convert_hf_to_gguf.py

```diff
@@ -7581,6 +7682,132 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iter
         return [(self.map_tensor_name(name), data_torch)]
 
 
+class MistralModel(TextModel):
```

Comment:
Maybe doing this will reduce the function to just `set_gguf_parameters` ?

```suggestion
class MistralModel(LlamaModel):
```

Reply:
Done